contracts/token/ERC777/ERC777.sol

```diff
@@ -103,7 +103,7 @@ contract ERC777 is Context, IERC777, IERC20 {
      *
      * This implementation always returns `1`.
      */
-    function granularity() public view virtual override returns (uint256) {
+    function granularity() public pure virtual override returns (uint256) {
```

Comment:
Granularity is defined as `view` so that a child contract can override this function with an implementation that reads from storage. As far as I can tell there is no benefit to use `pure` here.

Reply:
That makes sense.